### PR TITLE
Feature obsdef 7964 nginx grafana auth and UI loadbalancer

### PR DESCRIPTION
**Pull request by maheshkulkarni4**
1) https://jira.cec.lab.emc.com/browse/OBSDEF-7964 - Nginx config in ObjectScale Portal was updated with authentication for Grafana requests.

Auth request is sent to fedsvc 'validate' endpoint with X-SDS-Auth-Token header.
Token value is retrieved from OSAuthToken that will be provided by UI.

Additional config was added to handle logout from Grafana UI.
After user logs out from Grafana, it redirects to 'logout_grafana' endpoint in Nginx that issues request to fedsvc 'logout' endpoint and clears OSAuthToken cookie.

2) Enabled ObjectscaleSystems by default

3) https://jira.cec.lab.emc.com/browse/OBSDEF-9273 - Added LoadBalancer service for ObjectscalePortal for Serving Grafana services. 

## PR checklist
- [ ] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [x] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [ ] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
_Paste URL of charts-custom-ci job here_

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

**Review by gusakk — APPROVED** [review]

File: objectscale-portal/templates/objectscale-portal-configmap.yaml [hunk 1]
```diff
@@ -68,11 +85,80 @@ data:
...
+                # for now we set expiration time to 5m, later may be changed to use value from authsvc response header
+                proxy_cache_valid 5m;
+                # cut all redundant headers excluding those which are set in this location
+                proxy_pass_request_headers off;
+                proxy_pass_request_body off;
+                # take token value from cookie with name OSAuthToken and set it as header
+                proxy_set_header X-SDS-AUTH-TOKEN $cookie_osauthtoken;
+                proxy_set_header X-ALLOW-ROLES "SYSTEM_ADMIN,SYSTEM_MONITOR";
+                proxy_set_header Content-Length "";
+                proxy_pass https://$fedsvcname:$fedsvcport/mgmt/validate;
```
Comment (victor-polyansky) [inline objectscale-portal/templates/objectscale-portal-configmap.yaml:138]: Please change it to `proxy_pass http://$fedsvcname:$fedsvcport/mgmt/validateToken;` (missed to fix this in draft pr)

File: objectscale-portal/templates/objectscale-portal-configmap.yaml [hunk 2]
```diff
@@ -68,11 +85,80 @@ data:
...
+                internal;
+                # override global setting
+                proxy_buffering on;
+                # cut all redundant headers excluding those which are set in this location
+                proxy_pass_request_headers off;
+                proxy_pass_request_body off;
+                # take token value from cookie with name OSAuthToken and set it as header
+                proxy_set_header X-SDS-AUTH-TOKEN $cookie_osauthtoken;
+                proxy_set_header Content-Length "";
+                proxy_pass https://$fedsvcname:$fedsvcport/mgmt/logout;
```
Comment (victor-polyansky) [inline objectscale-portal/templates/objectscale-portal-configmap.yaml:151]: Please change it to `proxy_pass http://$fedsvcname:$fedsvcport/mgmt/logout;`

**Review by ben-schumacher — APPROVED** [review]

File: objectscale-portal/templates/objectscale-portal-configmap.yaml (hunk 1 above)
Comment (maheshkulkarni4) [reply objectscale-portal/templates/objectscale-portal-configmap.yaml]: done

File: objectscale-portal/templates/objectscale-portal-configmap.yaml (hunk 2 above)
Comment (maheshkulkarni4) [reply objectscale-portal/templates/objectscale-portal-configmap.yaml]: Done

**Pull request merged**